### PR TITLE
linked-events.swagger fixes and addition

### PR DESCRIPTION
**Pull request by JuliusSaa**
Fixed issues where "keywordList" fetch failed due to wrong types. 
Ex. expected object, string was sent.
Also added new field "accessible"

### CODE REVIEW

File: linked-events.swagger.yaml [hunk 1]
```diff
@@ -486,8 +486,17 @@ definitions:
         type: string
         description: Unique identifier for this keyword_set. These should be URIs identifying the source and the keyword_set itself, and preferably also well formed http-URLs pointing to more information about the keyword.
       name:
-        type: string
-        description: Name for this keyword_set. This should be human readable, such that it could be shown as label in UI
+        type: object
+        properties:
+          fi:
+            type: string
+            description: Keyword in Finnish
```
Comment (anttileppa) [inline linked-events.swagger.yaml]:
```suggestion
            description: Keyword set name in Finnish
```

File: linked-events.swagger.yaml [hunk 2]
```diff
@@ -486,8 +486,17 @@ definitions:
...
+          sv:
+            type: string
+            description: Keyword in Swedish
```
Comment (anttileppa) [inline linked-events.swagger.yaml]:
```suggestion
            description: Keyword set name in Swedish
```

File: linked-events.swagger.yaml [hunk 3]
```diff
@@ -486,8 +486,17 @@ definitions:
...
+          en:
+            type: string
+            description: Keyword in English
```
Comment (anttileppa) [inline linked-events.swagger.yaml]:
```suggestion
            description: Keyword set name in English
```

File: linked-events.swagger.yaml [hunk 4]
```diff
@@ -723,6 +732,9 @@ definitions:
       publisher:
         type: string
         description: Organization responsible for this event record.
+      accessible:
```
Comment (anttileppa) [inline linked-events.swagger.yaml]: What is this?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: linked-events.swagger.yaml (hunk 4 above)
Comment (JuliusSaa) [reply linked-events.swagger.yaml]: This is boolean value to tell if the event is easily accessible. 
Not necessarily unobstructed access, but easily accessed event.
At least Tavastia events uses this.

Comment (anttileppa) [reply linked-events.swagger.yaml]: Field is not supported on the upstream project: https://github.com/City-of-Helsinki/linkedevents. Could you contribute this feature to upstream before suggesting change to this swagger-file?

Unsupported features can break clients using other versions of API, so i cannot accept this change.

Comment (JuliusSaa) [reply linked-events.swagger.yaml]: Deleted 'accessible' field for now

**Review by anttileppa — APPROVED** [review]

**Pull request merged**